### PR TITLE
Create landing product page

### LINKED ISSUE #35: Create landing product page

### PR DESCRIPTION
**Pull request by MtheDV**
- Create product components
   - Create Header component
   - Create Footer component
   - Create Landing component
 - Make Button components EM based, so they can change sizes
 - Add 'flat' option to button
 - Add mobile/tablet/desktop breakpoints

Closes #35

### CODE REVIEW

File: src/components/Global/Button/index.tsx [hunk 1]
```diff
@@ -16,12 +16,14 @@ const getButtonClass = (type: ButtonType) => {
   if (type === 'primary') return styles.btnPrimary;
   if (type === 'secondary') return styles.btnSecondary;
   if (type === 'outline') return styles.btnOutline;
+  if (type === 'flat') return styles.btnFlat;
 };
 
 const getArrowClass = (type: ButtonType) => {
   if (type === 'primary') return styles.arrowPrimary;
   if (type === 'secondary') return styles.arrowSecondary;
   if (type === 'outline') return styles.arrowOutline;
```
Comment (Opeyem1a) [inline src/components/Global/Button/index.tsx:25]: we could use `||` to make these 1 line

File: src/components/Product/Product.module.scss [hunk 2]
```diff
@@ -0,0 +1,19 @@
+@import '../../styles/abstracts/variables';
```
Comment (Opeyem1a) [inline src/components/Product/Product.module.scss:1]: I reckon this file should go in `styles/pages/` - the goal is that non-component specific styling goes there

File: src/pages/index.tsx [hunk 3]
```diff
@@ -1,21 +1,19 @@
...
-    );
+    // Then redirect the user to the app pages
   }
+
   return (
     <>
-      <h1>Not logged in</h1>
-      <button onClick={() => signIn()}>Login</button>
+      <Header />
+      <Landing />
```
[the remark this answers is not in the document]
Comment (Opeyem1a) [reply src/pages/index.tsx:15]: Maybe `<LandingContent />` for now and we can revisit this later (?)

File: src/components/Global/Button/Button.module.scss [hunk 4]
```diff
@@ -10,10 +10,12 @@
   justify-content: space-between;
   align-content: space-between;
   align-items: center;
-  padding: 0.3rem 0.3rem 0.3rem 0.5rem;
+  padding: 0.3em 0.3em 0.3em 0.75em;
```
[the remark this answers is not in the document]
Comment (Opeyem1a) [reply src/components/Global/Button/Button.module.scss:13]: I'm not sure what the `('small', 'medium', 'large')` idea means, but are you saying that using `em`, if the font size in a div was big and I wanted a button inside that div, I'd have to style that button to have `font-size: 1rem;` to make it look normal?

**Review by Opeyem1a — CHANGES_REQUESTED** [review]

File: src/components/Global/Button/Button.module.scss (hunk 4 above)
[the remark this answers is not in the document]
Comment (MtheDV) [reply src/components/Global/Button/Button.module.scss:13]: that is what it would be yes

[the remark this answers is not in the document]
Comment (MtheDV) [reply src/components/Global/Button/Button.module.scss:13]: by providing the 'small', 'medium', 'large' sizing options as props we can avoid this hassle and just have that functionality within the button component

**Review by Opeyem1a — APPROVED** [review]

**Pull request merged**